docs/configuring_case.rst

```diff
@@ -22,22 +22,11 @@ We can use the following configuration to change the case to upper:
      constant_002 :
         case : 'upper'
 
-Conversley, rule entity_008 defaults to uppercase.
-We can use the following configuration to change the case to lower:
-
-.. code-block:: yaml
-
-   ---
-
-   rule :
-     entity_008 :
-        case : 'lower'
-
 Changing Multiple Case Rules
 ############################
 
 If there are a lot of case rules you want to change, you can use the global option to reduce the size of the configuration.
-For example, if we want to uppercase everything except the entity name, we could write the following configuration:
+For example, if you want to uppercase everything except the entity name, you could write the following configuration:
```

Comment:
It is interesting I use **we** instead of **you**

Reply:
Personally, I have never come across 'we' in any documentation or tutorial. In humble opinion it is better to use 'you' when the software is provided as is, without warranty of any kind.